docs/security/threat-model.md

```diff
@@ -0,0 +1,219 @@
+## Threat 11 - Attacker deploys workloads to namespaces that are  exempt from admission control
+An attacker is able to deploy workloads to Kubernetes namespaces that are exempt
+from the  admission controller configuration.
+
+**Mitigation**
+RBAC rights are strictly controlled
+
+**To do**
+I think most of the RBAC is not Kubewarden responsability. But we can help our users if we:
+- Warning them in our docs and *suggest* some minimum RBAC to be used.
+- Provide a policy which detect RBAC changes and **maybe** block them. Is this possible?
+
+
+## Threat ID 12 - Block rule can be bypassed due to missing match (e.g.  missing initcontainers)
+An attacker created a workload manifest which uses a feature of the Kubernetes
+API which  is not covered by the admission controller
+
+**Mitigation**
+All rules are reviewed and tested.
+
+**To do**
+We may came up with some tests to cover this rules reviews
+We  should carefully review a PR changing the rules in the policies deployment
+
+## Threat ID 13 - Attacker exploits bad string matching on a blocklist to  bypass rules
+An attacker, who has rights to create workloads, bypasses a rule by exploiting
+bad string  matching.
+
+**Mitigation**
+All rules are reviewed and tested.
+
+**To do**
+We may came up with some tests to cover this rules reviews
+We  should carefully review a PR changing the rules in the policies deployment
+
+## Threat ID 14 - Attacker uses new/old features of the Kubernetes API  which have no rules
+An attacker, with rights to create workloads, uses new features of the Kubernetes
+API (for  example a changed API version) to bypass a rule.
+
+**Mitigation**
+All rules are reviewed and tested.
+
+**To do**
+We may came up with some tests to cover this rules reviews and API versions.
+We can create a configuration to reject by default requests where the API
+version not cover by the policy.  We should warning policies developers to cover
+all the supported API version in theirs tests and reject all of others.
+
+
+## Threat ID 15 - Attacker deploys privileged container to node running  Webhook controller
+An attacker, who has rights to deploy privileged containers to the cluster, creates
+a privileged  container on the cluster node where the admission controller webhook operates.
+
+**Mitigation**
+Admission controller uses restrictive policies to prevent privileged  workloads
+
+**To do**
+I do not know if Kuberwarden can help on this. Kubewarden does not have access to
+containers running in the cluster node.
+
+
+## Threat ID 16 - Attacker mounts a privileged node hostpath allowing  modification of Webhook controller configuration
+An attacker, who has rights to deploy hostPath volumes with workloads, creates a
+volume  which allows for access to the admission controller pod’s files.
+
+**Mitigation**
+Admission controller uses restrictive policies to prevent privileged  workloads
+
+**To do**
+We can add a recommended policy in the `kubewarden-default` Helm chart to prevent this.
```

Comment:
Avoid usage of first-person single/plural terms

Reply:
Done in https://github.com/kubewarden/docs/pull/119/commits/b0aa443bb8c68ec6ca9d017c505eff10ba291d5d